src/components/full-movie-card/full-movie-card.jsx

```diff
@@ -1,26 +1,40 @@
-import React, {useMemo} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import {Link, generatePath} from 'react-router-dom';
+import {Link, Redirect, generatePath} from 'react-router-dom';
+import {connect} from 'react-redux';
 
 import {MainPath} from '../../constants/paths';
 import PosterSize from '../../constants/poster-size';
-import {getAlikeMovies} from '../../utils/movie-util';
+import OperationStatus from '../../constants/operation-status';
+import {selectAlikeMovies, selectMovieById} from '../../store/selectors';
 
 import Logo from '../logo/logo';
 import MoviesList from '../movies-list/movies-list';
 import UserBlock from '../user-block/user-block';
 import MovieDescription from '../movie-description/movie-description';
 import MovieBackground from '../movie-background/movie-background';
 import MovieInfo from './movie-info';
+import SpinnerLoading from '../spinner-loading/spinner-loading';
+import Maintenance from '../maintenance/maintenance';
 
-const FullMovieCard = ({movies = [], movie = {}}) => {
+const FullMovieCard = ({moviesStatus, movie = {}, alikeMovies = []}) => {
```

Comment:
Вместо того, чтобы обрабатывать movie как пустой объект, проще сделать его обязательным. Заметил такое в нескольких местах.
```suggestion
const FullMovieCard = ({moviesStatus, movie, alikeMovies = []}) => {
```

Reply:
До лекции про тестирование предлагаю оставить как есть.